Resources/public/js/views/ezsearch-searchlistview.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ezsearch-searchlistview', function (Y) {
+    "use strict";
+  
+    Y.namespace('eZSearch');
+
+    Y.eZSearch.SearchListView = Y.Base.create('ezsearchSearchListView', Y.eZ.TemplateBasedView, [], {
+      
+        render: function () {
+            this._setResultItems();
+            this.get('container').setHTML(this.template({
+                searchResultItems: this.get('searchResultItems'),
+                searchResultCount: this.get('searchResultCount'),
+            }));
+            return this;
+        },
+        
+        _setResultItems: function () {
+            var searchResultItems = [];
+
+            if ( !this.get('searchResultList') ) {
+                return this.get('searchResultList');
+            }
+
+            this.get('searchResultList').forEach(function (struct) {
+                searchResultItems.push({
+                    content: struct.content.toJSON(),
+                    contentType: struct.contentType.toJSON(),
+                    location: struct.location.toJSON(),
+                });
+            }, this);
+            this._set('searchResultItems', searchResultItems);
```

Comment:
I still don't see the point of this attribute, just change that method to be called `_getResultItems` and to return the data ready for the template, and then in `render`, you can directly use it's return value.